app/src/main/java/org/oppia/app/home/recentlyplayed/RecentlyPlayedActivityPresenter.kt

```diff
@@ -0,0 +1,24 @@
+package org.oppia.app.home.recentlyplayed
+
+import androidx.appcompat.app.AppCompatActivity
+import org.oppia.app.R
+import org.oppia.app.activity.ActivityScope
+import javax.inject.Inject
+
+/** The presenter for [RecentlyPlayedActivity]. */
+@ActivityScope
+class RecentlyPlayedActivityPresenter @Inject constructor(private val activity: AppCompatActivity) {
+  fun handleOnCreate() {
+    activity.setContentView(R.layout.recently_played_activity)
+    if (getContinuePlayingFragment() == null) {
+      activity.supportFragmentManager.beginTransaction().add(
+        R.id.recently_played_fragment_placeholder,
+        RecentlyPlayedFragment()
+      ).commitNow()
+    }
+  }
+
+  private fun getContinuePlayingFragment(): RecentlyPlayedFragment? {
```

Comment:
Please change this method to getRecentlyPlayedFragment()

Reply:
Done